realm-jni/src/util.hpp

```diff
@@ -19,6 +19,7 @@
 
 #include <string>
 #include <sstream>
+#include <memory>
```

Comment:
I think it's because now we use the unique pointers from the standard library instead of our own